app/src/python/load-wheels.py

```diff
@@ -1,17 +1,27 @@
 # pylint: disable=import-error
 
-
-import hashlib
+# import sys
 import importlib
 import io
 import json
 from pathlib import Path
 import zipfile
 
-from distlib import markers, util, version
+from js import Promise, XMLHttpRequest
+
+# sys.setrecursionlimit(10000)
+
 
-from js import Promise, window, XMLHttpRequest, pyodide as js_pyodide
-import pyodide
+WHEEL_BASE_PATH = Path('/lib/python3.7/site-packages/').parent
+VENDOR_WHEEL_BASE_URL = 'https://pyodide.pymedphys.com/wheels/'
+VENDOR_WHEEL_INDEX_FILENAME = 'index-7dc273d8-32cf-565f-8a3d-2f8ff86dde8d.json'
```

Comment:
All wheels are now hosted at https://pyodide.pymedphys.com/wheels/ I have changed the caching rules to tell the browser to save all the files for the next 2 months. ... that way, hopefully page load times will now be much, much quicker.

Reply:
Cool!